package.json

```diff
@@ -77,15 +78,17 @@
     "webpack-bundle-analyzer": "4.5.0",
     "webpack-cli": "4.10.0",
     "webpack-dev-server": "4.7.3",
-    "webpack-merge": "5.8.0"
+    "webpack-merge": "5.8.0",
+    "@fullhuman/postcss-purgecss": "^5.0.0"
   },
   "devDependencies": {
     "@babel/preset-typescript": "^7.18.6",
     "@types/jest": "^26.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.11",
     "ts-jest": "^26.5.0",
-    "typescript": "^4.9.4"
+    "typescript": "^4.9.4",
+    "purgecss-webpack-plugin": "^5.0.0"
```

Comment:
Needs alphabetical ordering here as well.

Reply:
Good catch, I think I should remove it alotgather, it was used in the first draft then I switch to @fullhuman/..etc